pom.xml

```diff
@@ -245,6 +245,32 @@
       <version>1.0</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-api</artifactId>
+      <version>1.39.0</version>
+    </dependency>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-exporter-logging</artifactId>
+      <version>1.39.0</version>
+    </dependency>
+    <dependency>
+      <groupId>io.opentelemetry.semconv</groupId>
+      <artifactId>opentelemetry-semconv</artifactId>
+      <version>1.25.0-alpha</version>
+    </dependency>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-exporter-zipkin</artifactId>
+      <version>1.39.0</version>
+    </dependency>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-exporter-otlp</artifactId>
+      <version>1.39.0</version>
+    </dependency>
```

Comment:
Can you change all of them, except the first one, to provided? And add them without provided to the test webapp? I want to keep the number of dependencies that Ladybug adds to other applications to a minimum.

Reply:
Created pr https://github.com/wearefrank/ladybug-test-webapp/pull/22. I will change it to provided when this pr is merged

Reply:
Done